### ISSUE TITLE
sqitch_mysql failed to install in mac os 

### ISSUE THREAD

**Issue by pengfei-xue** [sqitchers/homebrew-sqitch#16]
mac os 10.10.5

install with homebrew, got an error when install it, can you help to resolve this?

```
==> Installing sqitch_mysql from theory/homebrew-sqitch
==> Installing dependencies for theory/sqitch/sqitch_mysql: sqitch_dependencies, sqitch
==> Installing theory/sqitch/sqitch_mysql dependency: sqitch_dependencies
==> Downloading http://api.metacpan.org/source/DWHEELER/App-Sqitch-0.9993/META.json
Already downloaded: /Library/Caches/Homebrew/sqitch_dependencies-0.9993.json
==> cpanm --local-lib '/usr/local/Cellar/sqitch_dependencies/0.9993' --notest Module::Build
==> cpanm --local-lib '/usr/local/Cellar/sqitch_dependencies/0.9993' --notest Module::Build
==> cpanm --local-lib '/usr/local/Cellar/sqitch_dependencies/0.9993' --notest Class::XSAccessor
==> cpanm --local-lib '/usr/local/Cellar/sqitch_dependencies/0.9993' --notest Pod::Simple
==> cpanm --local-lib '/usr/local/Cellar/sqitch_dependencies/0.9993' --notest Type::Tiny::XS
--> Working on Type::Tiny::XS
Fetching http://www.cpan.org/authors/id/T/TO/TOBYINK/Type-Tiny-XS-0.012.tar.gz ... OK
Configuring Type-Tiny-XS-0.012 ... OK
Building Type-Tiny-XS-0.012 ... ! Installing Type::Tiny::XS failed. See /private/tmp/sqitch_dependencies20150920-1120-1sojkfg/.brew_home/.cpanm/work/1442745019.2032/build.log for details. Retry with --force to force install it.
FAIL

READ THIS: https://git.io/brew-troubleshooting
If reporting this issue please do so at (not Homebrew/homebrew):
  https://github.com/theory/homebrew-sqitch/issues
```

**Comment by pengfei-xue** [comment]
BTW, nothing found in

     /private/tmp/sqitch_dependencies20150920-1120-1sojkfg/.brew_home/.cpanm/work/1442745019.2032/build.log

**Comment by theory** [comment]
What does this output?

    cpanm --local-lib '/usr/local/Cellar/sqitch_dependencies/0.9993' --notest Type::Tiny::XS

**Comment by pengfei-xue** [comment]
`cpanm --local-lib '/usr/local/Cellar/sqitch_dependencies/0.9993' --notest Type::Tiny::XS` this can be run successfully in my mac box.

**Comment by theory** [comment]
Hrm. Well then does the Sqitch install work now?

**Comment by meticulo3366** [comment]
To Solve this issue I used Perl, hopefully my work around can lead to a proper fix in the `Brew` workflow..

`sudo cpan local::lib`
`sudo cpan App::Sqitch DBD::Pg`

**Comment by pengfei-xue** [comment]
@theory i installed it with cpan.

**Comment by meticulo3366** [comment]
The following might solve the problem..

`sudo cpanm --quiet --notest App::Sqitch`
`sudo cpan local::lib`
`sudo cpan App::Sqitch DBD::Pg`

**Comment by theory** [comment]
Try the new formula, released last night for v0.9998:

```
brew install sqitch --with-postgres-support --without-postgresql
```
